airbyte-cdk/java/airbyte-cdk/core/src/test/kotlin/io/airbyte/cdk/integrations/base/AirbyteTraceMessageUtilityTest.kt

```diff
@@ -48,8 +48,18 @@ class AirbyteTraceMessageUtilityTest {
             Mockito.mock(RuntimeException::class.java),
             "this is a config error"
         )
-        val outJson = Jsons.deserialize(outContent.toString(StandardCharsets.UTF_8))
-        assertJsonNodeIsTraceMessage(outJson)
+        val outCt = outContent.toString(StandardCharsets.UTF_8)
+        var outJson: JsonNode? = null
+        // because we are running tests in parallel, it's possible that another test is writing to
+        // stdout while we run this test, in which case we'd see their messages.
+        // we filter through the messages to find an error (hopefully hours)
+        for (line in outCt.split('\n')) {
+            if (line.contains("\"error\"")) {
```

Comment:
:cry:

it looks like we're expecting a json blob - can we filter for something like `"type":"trace"`? or maybe filter for lines that are valid json

(otherwise wouldn't this trigger for any `logger.warn("error while blah blah")` message?)